Source/hydro/trace_ppm.cpp

```diff
@@ -459,6 +477,10 @@ Castro::trace_ppm(const Box& bx,
     }
 
 
+    // for well-balanced, the velocity sources should not be added
+
+    amrex::Real fac = (castro::ppm_well_balanced && in_hse) ? 0.0_rt : 1.0_rt;
```

Comment:
is it better to just skip the source trace tracing for `un` above if we use `ppm_well_balaneced` and `in_hse`?

Reply:
I guess so, but somehow I thought this was more clear when reading the code.